solana/program_v2/packages/tests/src/pass-suite/change_pass_gatekeeper.ts

```diff
@@ -1,12 +1,12 @@
 import { createGatekeeperService, createNetworkService } from './util';
-import { GatekeeperService } from '@identity.com/gateway_v2-client/src/GatekeeperService';
+import { GatekeeperService } from '@identity.com/gateway-solana-client/src/GatekeeperService';
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
-import { NetworkService } from '@identity.com/gateway_v2-client/src/NetworkService';
+import { NetworkService } from '@identity.com/gateway-solana-client/src/NetworkService';
 import { Keypair, PublicKey } from '@solana/web3.js';
 import { TEST_ALT_NETWORK, TEST_NETWORK } from '../util/constants';
 import { setGatekeeperFlags } from '../util/lib';
-import { GatekeeperKeyFlags } from '@identity.com/gateway_v2-client/src/lib/constants';
+import { GatekeeperKeyFlags } from '@identity.com/gateway-solana-client/src/lib/constants';
```

Comment:
Just to note again.. this should be exported directly, and not referenced in the folder.. dbl check if the other PR exports everything needed